src/components/directorybrowser/directorybrowser.js

```diff
@@ -61,10 +57,6 @@ function refreshDirectoryBrowser(page, path, fileOptions, updatePathOnError) {
                 html += getItem(cssClass, folder.Type, folder.Path, folder.Name);
             }
 
-            if (!path) {
-                html += getItem('lnkPath lnkDirectory', '', 'Network', globalize.translate('ButtonNetwork'));
```

Comment:
What about the translation?

Reply:
Good call. I thought it might have been used in other places, but it is not. :slightly_smiling_face: